drv/opentitan-aontimer/src/lib.rs

```diff
@@ -0,0 +1,150 @@
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at https://mozilla.org/MPL/2.0/.
+
+#![no_std]
+
+#[allow(unused_imports)]
+use userlib::*;
+// use zerocopy::AsBytes;
+
+use core::ptr::slice_from_raw_parts_mut;
+
+const WDOG_REGWEN: usize = 4;
+const WDOG_CTRL: usize = 5;
+const WDOG_BARK_THOLD: usize = 6;
+const WDOG_BITE_THOLD: usize = 7;
+const WDOG_COUNT: usize = 8;
+const INTR_STATE: usize = 9;
+
+// The OpenTitan Always-On Timer is a counter that increments. If the counter
+// reaches a configurable threshold, it will emit an interrupt signal, called the
+// "bark". If the counter reaches a second configurable threshold, it will reset
+// the system, called the "bite".
+// Enabling the AON Timer before setting a bite threshold causes the system to
+// immediately reset.
+pub struct AonTimer {
+    base_addr: *mut [u32],
+    clock_freq: u32,
```

Comment:
Use suffix _hz in the name so that it is clear that the clock frequency is stored in Hz?

Reply:
Done.